src/FlashMessages.php

```diff
@@ -183,4 +193,14 @@ public function prepareMessages(SessionInterface $session, string $sessionKey):
 
         $this->currentMessages = $currentMessages;
     }
+
+    /**
+     * @return StoredMessages
+     */
+    private function getStoredMessages(?string $sessionKey = null): array
+    {
+        /** @var StoredMessages|null $messages */
+        $messages = $this->session->get($sessionKey ?? $this->sessionKey, []);
+        return ! is_array($messages) ? [] : $messages;
```

Comment:
Unsure if relevant, but we may return `$message ?? []` here

Reply:
Nice catch.